_test/test_mpi_wrappers/test_parallel_config_and_clusters_factory.m

```diff
@@ -119,81 +162,125 @@ function test_parallel_config(obj)
             try % current herbert cluster can not have 'Default' configuration
                 pc.cluster_config = 'default';
             catch Err
```

Comment:
Very minor thing, but you're using a mix of `Err` and `ME` throughout

Reply:
yes, depending on the mood. But its everywhere.